phoebe/frontend/default_bundles/default_binary.bundle

```diff
@@ -144,7 +144,7 @@ null
 "kind": "star",
 "context": "component",
 "description": "Critical (maximum) value of the equivalent radius for the given morphology",
-"value": 2.0132751765376384,
+"value": 2.013275176537638,
```

Comment:
all the changes to this file are inconsequential; I'd leave the original version.